### ISSUE TITLE
Unable to set movie format, MovieFormat: Produces invalid file names whatever I enter

### ISSUE THREAD

**Issue by foux** [Radarr/Radarr#10560]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

I'm trying to update my existing (and working) movie format, which is now : 
`{Movie OriginalTitle} {Edition Tags} {[Quality Full]}{[MediaInfo 3D]}{[MediaInfo VideoDynamicRange]}{[MediaInfo VideoDynamicRangeType]}[{Mediainfo VideoBitDepth}bit][{Mediainfo VideoCodec}]{[Mediainfo AudioCodec}{ Mediainfo AudioChannels}{ MediaInfo AudioLanguages}]{MediaInfo SubtitleLanguages:EN+FR}{[Custom Formats]}{-Release Group}`

But whatever I enter, when I try to save, I've got nothing in the UI (and I've got this in the logs : `MovieFormat: Produces invalid file names`
Even if I enter something as simple as `{Movie OriginalTitle}` with nothing else

As seen in the log, it tries to parse `The Original Movie Title` which is nowhere in my string

### Expected Behavior

My new file name format would save and work

### Steps To Reproduce

1. Go to the MediaManagement page
2. Enter a new Movie Format

### Environment

```markdown
- OS: Unraid
- Radarr: 5.13.0.9348
- Docker Install: Yes
- Using Reverse Proxy: Yes
- Browser: Chrome
- Database: Postgres
```

### What branch are you running?

Develop

### Trace Logs? **Not Optional**

2024-10-17 15:31:21.9|Debug|Radarr.Http.Authentication.ApiKeyAuthenticationHandler|AuthenticationScheme: API was successfully authenticated.
2024-10-17 15:31:21.9|Debug|MediaInfoFormatter|Unknown video format: 'AVC' in 'The.Movie.Title.2010.1080p.BluRay.DTS.x264-EVOLVE'. Streams: 
2024-10-17 15:31:21.9|Debug|MediaInfoFormatter|Unknown audio format: 'DTS' in 'The.Movie.Title.2010.1080p.BluRay.DTS.x264-EVOLVE'. Streams: 
2024-10-17 15:31:21.9|Debug|Parser|Parsing string 'The Original Movie Title'
2024-10-17 15:31:21.9|Debug|Parser|Unable to parse The Original Movie Title
2024-10-17 15:31:21.9|Trace|RadarrErrorPipeline|Handling Exception
2024-10-17 15:31:21.9|Trace|ConfigService|Using default config value for 'autounmonitorpreviouslydownloadedmovies' defaultValue:'False'
2024-10-17 15:31:21.9|Warn|RadarrErrorPipeline|Invalid request Validation failed: 
 -- MovieFormat: Produces invalid file names
2024-10-17 15:31:21.9|Trace|ConfigService|Using default config value for 'rsssyncinterval' defaultValue:'30'
2024-10-17 15:31:21.9|Trace|ConfigService|Using default config value for 'listsynclevel' defaultValue:'disabled'
2024-10-17 15:31:21.9|Trace|Http|Res: 261 [PUT] /api/v3/config/naming: 400.BadRequest (6 ms)
2024-10-17 15:31:21.9|Trace|ConfigService|Using default config value for 'maximumsize' defaultValue:'0'
2024-10-17 15:31:21.9|Trace|ConfigService|Using default config value for 'preferindexerflags' defaultValue:'False'
2024-10-17 15:31:21.9|Debug|Api|[PUT] /api/v3/config/naming: 400.BadRequest (6 ms)

### Trace Logs have been provided as applicable. Reports will be closed if the required logs are not provided.

- [X] I have read and followed the steps in the wiki link above and provided the required trace logs - the logs contain `trace` - that are relevant and show this issue.

**Comment by foux** [comment]
No, I didn't put the year in the file name, is that required?
But indeed the file with which Radarr is parsing is 2001 A space odissey. Is it possible to change the reference movie for that page?

**Comment by bakerboy448** [comment]
Given year is required for all parsing - pretty sure you'll need it in the movie name as well.

**Comment by foux** [comment]
It works by adding the release year, thanks a lot @bakerboy448. Is that a new requirement?

**Comment by bakerboy448** [comment]
No it's been a requirement, but validation was perhaps improved.

@mynameisbogdan any easy messaging to improve here, otherwise can close as support

**Comment by foux** [comment]
> Most likely was fixed in [1d286df](https://github.com/Radarr/Radarr/commit/1d286df85d3741c4047e5f23a20508edfcc6e64f).
> 
> I'll check later if there's anything to improve with the messaging, but most likely it was because it wasn't parsable due to lack of year.

I agree that's most likely the issue, but there's litteraly 0 feedback on the UI, except for the save button that stays highlighted.
